extensions/eclipse-che-theia-plugin-ext/src/browser/plugin/che-plugin-menu.ts

```diff
@@ -33,12 +34,26 @@ export class ChePluginMenu {
 
     protected readonly menuClosed = new Emitter<void>();
 
+    /**
+     * TEMPORARY SOLUTION
+     *
+     * Following code removes 'View/Plugins' menu item and the command that displays/hides Plugins view.
+     * In the future we will try to refactor Che Plugins view and move it to the 'plugin-ext'.
+     */
+    constructor(
+        @inject(MenuModelRegistry) menuModelRegistry: MenuModelRegistry,
+        @inject(CommandRegistry) commandRegistry: CommandRegistry
+    ) {
+        menuModelRegistry.unregisterMenuAction('pluginsView:toggle', CommonMenus.VIEW_VIEWS);
```

Comment:
couldn't we just unbind the stuff from inversify module ?

Reply:
Unbinding of PluginFrontendViewContribution does not work with failing Theia. I noticed that in a title.
And yes, it's an easiest and most proper way to unregister.

Reply:
I have played with it a lot yesterday.
The problem that `PluginFrontendViewContribution` is binded as `Command`, `Keybinding` and as `Menu` contribution.
I don't know how we can unbind the contribution from our extension.